### PR TITLE
Disable systemd-resolved

### PR DESCRIPTION
**Pull request by edsantiago**
We're pretty certain that it's the cause of the cdn03.quay.io flakes

Signed-off-by: Ed Santiago <santiago@redhat.com>

### CODE REVIEW

File: systemd_banish.sh [hunk 1]
```diff
@@ -41,3 +41,44 @@ if [[ -d "$EAAD" ]]; then
...
+# Opinions differ on the merits of systemd-resolve, but the fact is
+# it breaks our CI testing. Here we disable it for all VMs.
+# shellcheck disable=SC2154
+if ! ((CONTAINER)); then
+    nsswitch=/etc/authselect/nsswitch.conf
+    if [[ -e $nsswitch ]]; then
+        if grep -q -E 'hosts:.*resolve' $nsswitch; then
+            echo "Disabling systemd-resolved"
+            $SUDO sed -i -e 's/^\(hosts: *\).*/\1files dns myhostname/' $nsswitch
+            $SUDO systemctl stop systemd-resolved
```
Comment (cevich) [inline systemd_banish.sh:61]:
```suggestion
            $SUDO systemctl disable --now systemd-resolved
```

**Review by cevich — COMMENTED** [review]
LGTM overall, just one small change is needed.  Thanks for opening this, I was about to do it myself then noticed your PR :grin:

File: systemd_banish.sh (hunk 1 above)
Comment (edsantiago) [reply systemd_banish.sh]: Thank you! Fixed and repushed.

**Comment by cevich** [comment]
Need a new IMG_SFX file (one per push) also.

**Comment by cevich** [comment]
Reviewed logs for fedora images and found `Disabling systemd-resolved`.  I don't see this string for debian, but I'm assuming systemd-resolved isn't used there.

**Comment by cevich** [comment]
@edsantiago I tagged this version, so it will automatically roll-out to repos. with Renovate enabled.  [Renovate support in Podman](https://github.com/containers/podman/pull/18044) is a WIP but I expect merge this week or next.  Otherwise if you don't want to wait, you're welcome to open image-update PRs manually.  Thanks again for taking care of this.

**Comment by edsantiago** [comment]
Thanks @cevich. Sounds like a good test of Renovatebot.

**Pull request merged**